card/lib/card/migration.rb

```diff
@@ -28,8 +27,12 @@ def find_unused_name base_name
       test_name
     end
 
-    def paths mig_type=type
-      Card.paths["db/migrate#{schema_suffix type}"].to_a
+    def migration_paths mig_type=type
+      Cardio.migration_paths mig_type
+    end
```

Comment:
The wikirate comment didn't seem to be anchored to code.  I think you mean this method.  I think this is a necessary rename.  All the other places, 'paths' is a rails Paths object, and this isn't.  It's an array, I think.  And it may be loaded from the other paths configuration.

And for sure we should make sure I'm not breaking any API we use unless we are making a change we can all agree to.